kubernetes/cache/kube_cache.go

```diff
@@ -1464,15 +1464,27 @@ func (c *kubeCache) GetTelemetries(namespace, labelSelector string) ([]*v1alpha1
 	return retTelemetries, nil
 }
 
+func (c *kubeCache) IsK8sGatewayListerInit(namespace string) bool {
```

Comment:
Can this be made private?  Not sure it needs to be Capitalized?

Reply:
agree, I was trying to use that method from `func (in *IstioConfigService) getIstioConfigListForCluster` but it was a bad idea.